### ISSUE TITLE
Don't add [meta] back to a bug title if it has just been removed

### ISSUE THREAD

**Issue by Standard8** [mozilla/relman-auto-nag#958]
STR:

* Remove [meta] from the title of the bug (because it was never used as a meta, but is reasonable to consider as a future enhancement).
* Forget about changing the keyword field.

Expected Results

* Nag bot removes the keyword.

Actual Results

* Nag bot adds [meta] back to the title again.

**Comment by marco-c** [comment]
> Nag bot removes the keyword.

Maybe we should needinfo instead to let a human remove the inconsistency (what if the `[meta]` was removed in error?).

**Comment by calixteman** [comment]
We can probably get the bug history and check if meta has been removed from the title by a human and so remove the keyword if present.
@Standard8 do you have an example of such bug ?

**Comment by Standard8** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=592479 was the one I saw this in.

**Comment by marco-c** [comment]
@Standard8 the rule is to add "[meta]" to the title when there is the "meta" keyword. If a bug isn't a meta, you should remove both "[meta]" from the title and "meta" from the keywords.
Perhaps the bot should be more clear and add a comment saying something like "[meta] added in the title because the keywords contain 'meta'" so people know what to do.

**Comment by Standard8** [comment]
> @Standard8 the rule is to add "[meta]" to the title when there is the "meta" keyword. If a bug isn't a meta, you should remove both "[meta]" from the title and "meta" from the keywords.

That's fine and I understand the rule, but if I've literally just removed `[meta]` from one of the places, couldn't the bot detect that and remove it from the other place, rather than just adding it back again?

**Comment by marco-c** [comment]
Yeah, we could also do that, though it's a more invasive change (and might not be so trivial if there's more than one meta-related change to the keywords/title).
